spec.json

```diff
@@ -1054,19 +1064,6 @@
                             ]
                         }
                     ]
-                },
-                {
-                    "propertyName": "Color",
-                    "values": [
-                        { "name" : "NoColor",  "value" : "0", "type" : "int", "description": [ "No color." ] },
-                        { "name" : "Black",    "value" : "1", "type" : "int", "description": [ "Black color." ] },
-                        { "name" : "Blue",     "value" : "2", "type" : "int", "description": [ "Blue color." ] },
-                        { "name" : "Green",    "value" : "3", "type" : "int", "description": [ "Green color." ] },
-                        { "name" : "Yellow",   "value" : "4", "type" : "int", "description": [ "Yellow color." ] },
-                        { "name" : "Red",      "value" : "5", "type" : "int", "description": [ "Red color." ] },
-                        { "name" : "White",    "value" : "6", "type" : "int", "description": [ "White color." ] },
-                        { "name" : "Brown",    "value" : "7", "type" : "int", "description": [ "Brown color." ] }
-                    ]
```

Comment:
I would be very grateful if you could do it for me please.  I'm obviously a bit out of my depth and don't have the luxury of time at the moment to get a good handle on it.  Are you also able to do this for the ev3dev.h and ev3dev.cpp ones?